packages/s/snapd/files/wrapper.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+YELLOW='\033[0;33m'
+NC='\033[0m'
+URL=https://help.getsol.us/docs/user/software/third-party/snap
```

Comment:
Some vars are quoted, some are not? Is this on purpose...?

Reply:
All quoted now